googlemaps/places.py

```diff
@@ -44,7 +44,7 @@
 
 PLACES_FIND_FIELDS_CONTACT = {"opening_hours"}
 
-PLACES_FIND_FIELDS_ATMOSPHERE = {"price_level", "rating", "user_ratings_total"}
+PLACES_FIND_FIELDS_ATMOSPHERE = {"editorial_summary","price_level", "rating", "user_ratings_total"}
```

Comment:
`editorial_summary` is not supported for Find Place. See [documentation](https://developers.google.com/maps/documentation/places/web-service/place-data-fields#places-api-fields-support).